article/detorakis-2016.tex

```diff
@@ -295,72 +295,101 @@ \section{Methods}\label{methods}
-to the one used by the author in the original article (the author has
-numerically integrated the system of the four ODEs by using a fifth-order
-adaptive size Runge-Kutta method). \emph{BDF} and \emph{Adams} provide
-exactly the same numerical results as the first one, but they are
-faster\footnote{The user has the option to choose one of the three methods.
-    Initially,
-    we tested all three methods and since there is no any numerical difference
-    on our results we decided to use \emph{BDF}, since its much faster than
-    \emph{dopri5}.}.
+Three different methods have been tested in this work (\emph{dopri5}, 
+\emph{Adams}, \emph{BDF}, \cite{ascher:1998}). \emph{dopri5} is the closest
+numerical method to the one used by the author in the original article (the
+author has numerically integrated the system of the four ODEs by using a
+fifth-order adaptive size Runge-Kutta method). \emph{BDF} and \emph{Adams}
+provide similar numerical results as the first one, but they are
+faster\footnote{The user has the option to choose one of the three methods
+    during class instantiating. We tested all three methods comparing, spike
+    times, amplitudes and the coefficient of variation using as threshold value
+    $0\, \Rm{mV}$. Spike events were overlapping, and $CV \simeq 1.5$ for all
+    three methods. The membrane potential amplitude was sometimes different
+    for the same spike events, however that difference was not too high as
+    Figure~\ref{Fig:1} shows (up to $4\, \Rm{mV}$ at maximum). Thus we decided
+    to use the \emph{Adams} method.}.
 %%
 \begin{table}[!htbp]
     \centering
     \begin{tabular}{ccc}
         \thickhline
         \multicolumn{3}{c}{Simulation Time} \\ \thickhline
-        Figure & Simulation Time ($s$) & Integration Step ($ms$) \\ \rowcolor{LightGray}
-        $1$ & $2$, $5\times period$ & $0.05$ \\ \rowcolor{Gray}
-        $2$ & $2$ & $0.05$ \\ \rowcolor{LightGray} 
-        $3$ & $8$ & $0.05$  \\ \rowcolor{Gray} 
-        $4$ & $2.5$ & $0.05$  \\ \rowcolor{LightGray}  
-        $5$ & $2$ & $0.05$  \\ \thickhline
+        Figure & Simulation Time ($\Rm{s}$) & Integration Step ($\Rm{ms}$) \\ 
+        \rowcolor{LightGray}
+        $1$ & $6$ & $0.05$ \\ \rowcolor{Gray}
+        $2$ & $15\times period$, $2$ & $0.05$ \\ \rowcolor{LightGray}
+        $3$ & $6$ & $0.05$ \\ \rowcolor{Gray} 
+        $4$ & $1.5$, $1.5$, $1$ & $0.05$ \\ \rowcolor{LightGray} 
+        $5$ & $5$ & $0.05$  \\ \rowcolor{Gray} 
+        $6$ & $2.5$ & $0.05$  \\ \rowcolor{LightGray} 
+        $7$ & $40$, $20$ & $0.05$  \\ \thickhline
     \end{tabular}
     \caption{{\bfseries \sffamily Simulations Time}}
     \label{Table:2}
 \end{table}
 %%  
 
+\begin{figure}[!htbp]
+    \centering
+    \includegraphics[width=0.6\textwidth]{figs/Figure1.pdf}
+    \caption{{\bfseries \sffamily Spike waveform.} Three integration methods:
+    Runge-Kutta $5$ (\emph{dopri5}), \emph{Adams}, and \emph{BDF} where tested
```

Comment:
I would add the line style information here already: "Runge-Kutta $5$ (\emph{dopri5}, solid line), \emph{Adams} (dashed), and \emph{BDF} (dash-dotted)."